cmd/livepeer/livepeer.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	devenv := flag.Bool("devenv", false, "Set to true to enable devenv")
 	controllerAddr := flag.String("controllerAddr", "", "Protocol smart contract address")
 	gasLimit := flag.Int("gasLimit", 0, "Gas limit for ETH transactions")
-	gasPrice := flag.Int("gasPrice", 4000000000, "Gas price for ETH transactions")
+	gasPrice := flag.Int("gasPrice", 0, "Gas price for ETH transactions")
```

Comment:
One of the reasons we hard-coded this number was that the gas estimator was really unreliable on Rinkeby. Will it be better on mainnet?

Reply:
Yah I'd be willing to run a build with this option as my main net node for a little while before releasing it as an update only to find out it consistently underestimates, if that's a better path for testing.

Reply:
I've been testing it on the NY community node.  So far it's been working well.